app/components/CustomLoader.vue

```diff
@@ -20,5 +12,14 @@
 <style lang="less" scoped>
 .loader {
   width: 64px;
+  animation: loading-animation 2s infinite;
 }
-</style>
+svg {
+ fill:#70A0AF;
+}
+@keyframes loading-animation {
+  50%{
+    opacity: 0;
+  }
+}
+</style>
```

Comment:
末尾改行（editorconfigには指定があるので一応）
```suggestion
</style>
```

Reply:
改行追加しました